config.m4

```diff
@@ -402,6 +402,10 @@ PHP_ARG_WITH(libmongoc, whether to use system libmongoc,
     AC_SUBST(MONGOC_ENABLE_SECURE_TRANSPORT, 0)
     AC_SUBST(MONGOC_ENABLE_COMMON_CRYPTO, 0)
 
+    dnl Secure Channel only applies to Windows
+    AC_SUBST(MONGOC_ENABLE_SECURE_CHANNEL, 0)
```

Comment:
No problem. Thanks for the heads up!

I started this PR before realizing that only libbson was updated. I'll revisit once the upstream ticket is resolved in libmongoc.